src/librawspeed/metadata/CameraMetaData.h

```diff
@@ -23,18 +23,21 @@
 #include "common/Common.h" // for uint32
 #include <map>             // for map
 #include <string>          // for string
+#include <tuple>           // for tuple
 
 namespace RawSpeed {
 
 class Camera;
 
+using CameraId = std::tuple<std::string, std::string, std::string>;
+
```

Comment:
> this needs to be as non-intrusive as possible.

I understand. This change and my similar `TiffID` can certainly coexist. It just wanted to point out that in the end they should probably be merged. I was actually thinking about introducing a 'global' CameraID type to be used for both but also wanted to keep it 'minimal'.